src/core/server/stacks/createComment.ts

```diff
@@ -93,11 +94,29 @@ const markCommentAsAnswered = async (
     return;
   }
 
-  // If we have experts and this reply is created by
-  // one of them, then this is an expert's answer.
-  if (story.settings.expertIDs.some(id => id === author.id)) {
-    // We need to mark this question as answered now.
-    // We can now remove the unanswered tag.
+  // - Comment must be a direct ancestor of a top level
+  //   comment (question) to be an answer. This will mean
+  //   we only have one ancestor in our ancestor list.
+  // - If we have experts and this reply is created by
+  //   one of them, then this is an expert's answer.
+  if (
+    comment.ancestorIDs.length === 1 &&
+    story.settings.expertIDs.some(id => id === author.id)
+  ) {
+    // Expert's answers are featured so they appear
+    // in the featured (answered) tab
+    await addTag(
+      mongo,
+      tenant,
+      comment.id,
+      revisionID,
+      author,
+      GQLTAG.FEATURED
+    );
```

Comment:
I'm doing the addTag now since we've confirmed this is a top level question/answer pair via the previous if check on the ancestors.

This prevents nested answers being listed as q/a pairs.